### PR TITLE
Add external link icon to app website URL

### PR DESCRIPTION
**Pull request by marians**
Adds the "open in new window" icon to the website URL on the app detail page.

I wish the `&nbsp;` wouldn't be necessary, but that's how we do it in other places, too, so I'm consistently dumb. here.

## Status quo 

image

## Change preview

image

### CODE REVIEW

File: src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx [hunk 1]
```diff
@@ -326,7 +326,8 @@ const AppDetail: React.FC<IAppDetailPageProps> = (props) => {
             <AppDetailPageOptionalValue value={props.website}>
               {(value) => (
                 <a href={value} target='_blank' rel='noopener noreferrer'>
-                  {value}
+                  {value}&nbsp;
```
Comment (axbarsan) [inline src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx:329]:
```suggestion
                  {value}{' '}
```

Comment (marians) [reply src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx:329]: Will this prevent the icon from breaking into a separate line if the URL is long?

Comment (axbarsan) [reply src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx:329]: I don't know about that, but `{' '}` is what we use in other parts of the code base

Comment (marians) [reply src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx]: ACK

Comment (oponder) [reply src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx]: `{' '}` is a normal space, so it won't prevent it from breaking into a separate line. `&nbsp;` is a non-breaking space, which will prevent it.

alternatively you can make some changes to the CSS to adjust how whitespace is handled. Would have to Google exactly what property though, never know it off the top of my head

**Comment by marians** [comment]
@axbarsan Can I ask for your help fixing the tests please? Is it safe to update all snapshots?

**Review by axbarsan — APPROVED** [review]

**Comment by marians** [comment]
@axbarsan I can't run `yarn test -u` because of https://gigantic.slack.com/archives/CQ465CFG8/p1623914610001900 - Any idea?

**Comment by axbarsan** [comment]
> @axbarsan I can't run `yarn test -u` because of [gigantic.slack.com/archives/CQ465CFG8/p1623914610001900](https://gigantic.slack.com/archives/CQ465CFG8/p1623914610001900) - Any idea?

Update node to v16+

**Pull request merged**